README.md

```diff
@@ -1,3 +1,9 @@
+![Sudokuru Logo](https://sudokuru.s3.amazonaws.com/goldLogoText.png)
+
+# The Official Cross-Platform Frontend for our Free Open Source Sudoku Project
+
+# 💻 Try our DEV site at: https://sudokuru.pages.dev/
+
```

Comment:
Would be nice to link to our itch.io page for 
Android, Linux, and Windows builds https://sudokuru.itch.io/sudokuru